Index: xterm/xterm.man
===================================================================
--- xterm.orig/xterm.man
+++ xterm/xterm.man
@@ -2165,8 +2165,9 @@
 from \fIdtterm\fP the operation number).
 The default value is
 .RS
-20,21,SetXprop,SetSelection
+1,2,3,4,5,6,7,8,9,11,13,14,18,19,20,21,GetSelection,SetSelection,SetWinLines,SetXprop
 .RE
+(i.e. no operations are allowed).
 .IP
 The names are listed below.  \fIxterm\fP ignores capitalization, but
 they are shown in mixed-case for clarity.  Where a number can be used
